src/v/raft/kvelldb/kvserver.cc

```diff
@@ -181,8 +181,11 @@ extract_peer(ss::sstring peer) {
         throw std::runtime_error(fmt::format("Could not parse peer:{}", peer));
     }
     int32_t n = boost::lexical_cast<int32_t>(parts[0]);
+    std::vector<ss::sstring> addres_parts;
```

Comment:
```suggestion
    std::vector<ss::sstring> address_parts;
```
Weirdly, libc++ seems to be missing the `string_view` constructor from iterators so
```
    std::vector<std::string_view> address_parts;
```
Doesn't work with clang.